recordio/file_reader.go

```diff
@@ -350,28 +350,53 @@ func readNextV2(r *FileReader) ([]byte, error) {
 	return returnSlice, nil
 }
 
-// TODO(thomas): we have to add an option pattern here as well
```

Comment:
Hi Thomas, 

I have implemented this option pattern to be able to set the buffer size. The default one is too big for my use case.